app/javascript/components/Pagination/Pagination.tsx

```diff
@@ -8,6 +8,7 @@ import './Pagination.css';
 interface PaginationProps {
   pathname: string
   totalPages: number
+  unboundedResults: boolean
```

Comment:
`unboundedResults` will be `true` only for Bing searches. [The component is documented here](https://designsystem.digital.gov/components/pagination/).